src/components/DataGridComponent.jsx

```diff
@@ -142,7 +142,7 @@ function DataGridComponent({reposModCount, setReposModCount, isContentExperiment
                     columns: {
                         columnVisibilityModel: {
                             nBooks: false,
-                            source: isContentExperiment ? false : true,
+                            source: isNormal ? true : false,
```

Comment:
This is the trick I've mentioned before. The value `isNormal` is true or false. The value of the ternary is true or false. In such cases you never need a ternary. In this case you just need `isNormal`.